backend/api/Migrations/20230811081831_AddAnalysisType.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Api.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddAnalysisType : Migration
```

Comment:
Maybe choose a name that explains the change a bit better. Like ChangeAnalysisTypeToEnum

Reply:
Done !